Update template_csv_parser.py

ventura-rivera opened the pull request:
added extra keywords to if-conditions (in add_fields) so that they include the following terms:
'preparationname',
'processname',
'prepname',

jling11 commented on csv_template_ingester/template_csv_parser.py:
```diff
@@ -233,10 +233,10 @@ def add_fields(keywords, names, units, systs, sys_dict, row):
             systm = add_datatype(systm, cell)
 
-        elif 'preparationstepname' == normalize(keywords[j]) or 'processstepname' == normalize(keywords[j]):
+        elif 'preparationstepname' == normalize(keywords[j]) or 'processstepname' == normalize(keywords[j]) or 'preparationname' == normalize(keywords[j]) or 'processname' == normalize(keywords[j]) or 'prepname' == normalize(keywords[j]) or 'procname' == normalize(keywords[j]):
```
This could be more elegantly phrased as:
if normalize(keywords[j]) in ['preparationstepname', 'processstepname', etc etc]:

jling11 replied:
Also, add a comment.

jling11 commented on csv_template_ingester/template_csv_parser.py:
```diff
@@ -233,10 +233,10 @@ def add_fields(keywords, names, units, systs, sys_dict, row):
             systm = add_preparation_step(systm, cell)
 
-        elif 'preparationstepdetail' == normalize(keywords[j]) or 'processstepdetail' == normalize(keywords[j]):
+        elif 'preparationstepdetail' == normalize(keywords[j]) or 'processstepdetail' == normalize(keywords[j]) or 'preparationdetail' == normalize(keywords[j]) or 'processdetail' == normalize(keywords[j]) or 'prepdetail' == normalize(keywords[j]) or 'procdetail' == normalize(keywords[j]):
```
same as above

ventura-rivera commented:
Hey Hutch. I have also added a test (it is on another PR) by expanding on the test_add_fields() function on the test_template_csv_parser.py file. Also, the Travis CI build failed because of a syntax error that I have fixed

ventura-rivera commented:
Sure! I will figure out how to do that now
